### ISSUE TITLE
X Keyboard module does not display the current layout properly

### ISSUE THREAD

**Issue by d125q** [jaagr/polybar#230]
My X11 keyboard is the following:
```text
      X11 Layout: us,mk
       X11 Model: pc105
     X11 Options: grp:win_space_toggle,compose:ralt
```

In other words, it switches between `us` and `mk` by pressing `S+SPC`.  However, polybar constantly displays `us` as the layout.

**Comment by jaagr** [comment]
I'll have a look at it

**Comment by jaagr** [comment]
fixed by 891ff6d, please verify

**Comment by d125q** [comment]
Thank you; it works!
